### ISSUE TITLE
Side navigation menu links lack sufficient color contrast

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/rockarch.org#51]
Contrast ratio for side navigation link text does not currently meet the WCAG 2.0 AA requirement. Either use different colors or make text larger.

[Color contrast checker](https://webaim.org/resources/contrastchecker/)
